wiki/en/Running-a-Server.md

```diff
@@ -150,21 +141,17 @@ Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s
 
 <figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus server setup window"></figure>
 
-### Make My Server Public
-
-By default, you will be running a Private Server and need to [read these instructions](#running-a-private-server) to have others connect to you in this mode.
+### The Directory list 
 
-When making your server public, you should see a confirmation message saying whether your server has registered successfully. If not, and you leave your server running, it will keep trying to register until a free slot becomes available.
+**None**: By default, you will not be connected to a directory and will be running in unregistered mode.  [Read these instructions](#running-an-unregistered-server) to have other people connect to your server in this mode.
 
+**Genre**: To allow other people to see your server on one of the built-in public directories, select your desired genre directory. You should see a confirmation message saying whether your server has registered successfully. If not, and you leave your server running, it will keep trying to register until a free slot becomes available.  
 
-### Genre
-
-Because there is a limit of 150 servers per Directory, you need to select which Directory Server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list). This will make your server appear in the relevant server list for clients.
-
+**Custom**: This sets your server to be a directory on which other servers can be listed. Note that when setting this mode, you may want also to specify the location of server list file (See "Options" below) unless you want to disconnect all listed servers when restarting Jamulus.  
```

Comment:
> **Custom**: This sets your server to be a directory on which other servers can be listed. 

Wrong.

Custom allows you to set a Custom Directory server address.  Nothing more than that.

"Registering" with localhost is what makes a server a directory.

Reply:
See https://github.com/jamulussoftware/jamuluswebsite/pull/677#discussion_r780814854 This seems a rather odd UX.

Reply:
Why?

No one wants to run as a Directory apart from a handful of people.

1. Most people running a server register with a Public Directory (Jamulus built in address) - they pick a Genre.

2. Those not wanting to register at all pick None.

3. The next most common group is those who are registering with a Custom Directory - they select Custom here (and put the Custom Directory server address in the "Custom Directory server address" field).

4. The least common group is those who are running Directories (both the Public and Custom ones -- same deal).  Of these, a tiny fraction will want to do it using the GUI (so a tiny fraction of a tiny fraction).
Just like from the command line, they use a Custom Directory server address of "localhost" or "127.0.0.1" -- because that's how we identify running as a Directory -- and, just like any other GUI server operator they'll select Custom for Directory and enter the Custom Directory server address.

Reply:
My point was simply one of basic usability, but if we take the view that it's rare enough not to matter (and we're putting the Directory docs into another place now anyway) then that's OK.